docs/how-to-guides/environment-setup-and-maintenance/installing-isis-via-anaconda.md

```diff
@@ -27,7 +27,7 @@ This installation guide is for ISIS users interested in installing ISIS (3.6.0)+
     export CONDA_SUBDIR=osx-64
     
     #Create a new conda environment to install ISIS in
-    conda create -n isis python=3.9
+    conda create -n isis python>=3.9
```

Comment:
I imagine It would be fine to not specify a python version at all?

Reply:
There was a previous issue regarding NOT setting the python version. Let me see if I can find it